modules/twinklexfd.js

```diff
@@ -714,6 +714,29 @@ Twinkle.xfd.setWatchPref = function twinklexfdsetWatchPref(pageobj, pref) {
 };
 
 Twinkle.xfd.callbacks = {
+	// Requires having the tag text (params.tagText) set ahead of time
+	autoEditRequest: function(pageobj, params) {
+		// Update the form
+		pageobj.getStatusElement().warn('Page protected, requesting edit');
+
+		var venue = utils.toTLACase(params.venue);
+
+		var talkTitle = new mw.Title(pageobj.getPageName()).getTalkPage().toText();
+		var editRequest = '\n\n== Edit request to complete ' + venue + ' nomination ==\n';
+		editRequest += '{{Edit protected|' + talkTitle + '|answered=no}}\n';
+		editRequest += 'I nominated ' + pageobj.getPageName() + ' at ' + venue + ' ([[' + params.discussionpage + '|nomination]]), but it is protected so I cannot tag the page. Please add:';
+		editRequest += '\n\n<nowiki>' + params.tagText + '</nowiki>\n\n';
+		editRequest += 'to the top of the page to complete the nomination. Thank you. ~~~~';
```

Comment:
This should be a (substituted) template on-wiki.

Reply:
I guess?  It seems pretty one-Twinkle-task specific, but I suppose it helps with for portability.  safesubsts are top five least-favorite things.

Reply:
Also, someone creating an XfD without using Twinkle could use it. 

> safesubsts are top five least-favorite things.

Ikr